pom.xml

```diff
@@ -104,20 +141,30 @@
         <version>3.3.3-sp.2-SNAPSHOT</version><!-- {x-version-update:google-cloud-spanner:current} -->
       </dependency>
 
+<!--      <dependency>-->
```

Comment:
@elharo we had to remove the google-cloud-shared-dependencies, since some versions did not work with our build.